documentation/misc/Building_a_network_skeleton.md

```diff
@@ -59,13 +61,34 @@ The table below gives an overview of how Lotus and its critical dependencies rel
 
 ## Setup
 
-1. Clone the [ref-fvm](https://github.com/filecoin-project/ref-fvm.git) repository.
+1. Create a tracking issue for this effort:
```

Comment:
@rjan90 : I created this as a setup step, partly based on what you did for nv24.  I was thinking to compromise though and just create one tracking issue with tasks for each of the dependent repo (rather than one issue per repo).